sources/packages/web/src/services/AuthService.ts

```diff
@@ -32,7 +32,7 @@ export default async function(
     console.log(`KC - init excp : ${excp}`);
   }
   keycloak.onTokenExpired = () => {
-    console.log("onTokenExpired");
+    store.dispatch("auth/logout");
```

Comment:
we dont have a story for auth/logout as yet so better to remove this one?

Reply:
The second "Technical task" in the story mentions it and we talked about it during our "dev planning" and that is why this line was added.
After the implementation of the story that you are working on, this will be no needed anymore because the entire "auth/logout" will be removed.

Reply:
So do you want me to remove it with the code i am checkin in
?

Reply:
Yes, if you did not update your PR yet. Otherwise, I will do it once I merge your changes.

Reply:
k please take care of it